charts/kube-prometheus-stack/values.yaml

```diff
@@ -1227,6 +1227,23 @@ kubeApiServer:
     # - targetLabel: __address__
     #   replacement: kubernetes.default.svc:443
 
+    ## Enable scraping Service Level Indicator (SLI) metrics (/metrics/slis)
```

Comment:
As I recall, SLI was introduced at different times for different components.
Please give me a few moments to confirm.

Reply:
Sorry, my memory was not at all good...

SLI was introduced in v1.26 for all components, but is in Alpha state and requires the feature gate `ComponentSLIs` to be enabled.
Starting from v1.27, it is in Beta and is enabled by default.

Reply:
If you can put the info into the comments here, it would be great!

Something like 

```suggestion
    ## Enable scraping Service Level Indicator (SLI) metrics (/metrics/slis) (requires Kubernetes 1.27+)
```

Reply:
Added comment.s
1.26 is also mentioned along with the need to enable feature-gate, but will be removed if not needed.